### PR TITLE
Modified onnx_to_nnir.py to take input and output dimensions 

### PR DESCRIPTION
**Pull request by LakshmiKumar23**

### CODE REVIEW

**Comment by kiritigowda** [comment]
@LakshmiKumar23 is this ready to be merged? has this passed all the tests?

**Comment by LakshmiKumar23** [comment]
Yes, I have tried it with a few models and you need to specify input and output dimensions just as in caffe_to_nnir.

File: model_compiler/python/onnx_to_nnir.py [hunk 1]
```diff
@@ -164,23 +171,26 @@ def onnx2ir(model, output_folder):
...
     # convert ONNX graph to IR graph and save
-    graph = onnx_graph_to_ir_graph(onnx_model.graph)
+    graph = onnx_graph_to_ir_graph(onnx_model.graph, input_dims, output_dims)
     graph.toFile(output_folder)
 
 def main():
-    if len(sys.argv) < 3:
-        print('Usage: python onnx2nnir.py <onnxModel> <nnirOutputFolder>')
+    if len(sys.argv) < 7:
+        print('Usage: python onnx2nnir.py <onnxModel> <nnirOutputFolder> --input-dims n,c,h,w --output-dims n,c,h,w')
```
Comment (kiritigowda) [inline model_compiler/python/onnx_to_nnir.py]: The input and output dims need to be optional and work by default like before. The new options need to be documented in the readme.

Comment (LakshmiKumar23) [reply model_compiler/python/onnx_to_nnir.py]: I have made the input and output dimensions to be optional and added it to the readme.

Comment (kiritigowda) [reply model_compiler/python/onnx_to_nnir.py]: Remove `--output-dims` option from the script. The script needs to use the `n' from the input, (c,h,w) needs to come from the model. Talk to @rrawther if you have any questions.

File: model_compiler/README.md [hunk 2]
```diff
@@ -29,7 +29,10 @@ MIVisionX allows hundreds of different [OpenVX](https://www.khronos.org/registry
 To convert an ONNX model into AMD NNIR model:
 
 ```
-% python onnx_to_nnir.py <model.onnx> <nnirModelFolder>
+% python onnx_to_nnir.py <model.onnx> <nnirModelFolder> [OPTIONS]
+OPTIONS:
+	--input_dims n,c,h,w
+	--output_dims n,c,h,w
```
Comment (kiritigowda) [inline model_compiler/README.md]: --output_dims - not required

File: model_compiler/python/onnx_to_nnir.py [hunk 3]
```diff
@@ -1,186 +1,179 @@
...
+		onnx_model = model
+	else:
+		raise TypeError("Model must be file path to .onnx file or onnx loaded model")
+	# convert ONNX graph to IR graph and save
+	graph = onnx_graph_to_ir_graph(onnx_model.graph)
+	graph.toFile(output_folder)
+
+def main():
+	if len(sys.argv) < 3:
+		print('Usage: python onnx2nnir.py <onnxModel> <nnirOutputFolder> --input_dims n,c,h,w (optional) --output_dims n,c,h,w (optional)')
```
Comment (kiritigowda) [inline model_compiler/python/onnx_to_nnir.py]: --output_dims n,c,h,w (optional) - not required

**Pull request closed without merging**